### PR TITLE
Instructions on how to use the data registry

### PR DESCRIPTION
**Pull request by pfrisella**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,99 @@
-# ds-data-registry
-A registry of data sources, categories, and organizations to use with Data Studio Community Connectors.
+# Data Studio Data Registry
+A registry of data sources, categories, and organizations to use with [Data Studio Community Connectors](https://developers.google.com/datastudio/connector/overview).
+
+The registry contains the following :
+- sources/ - Data source definitions and associated metadata.
+- categories.json - Metadata about data source categories.
+- organizations.json - Metadata about data source organizations.
```
Comment (diminishedprime) [inline README.md]: This might look better as a table

File: README.md [hunk 2]
```diff
@@ -1,2 +1,99 @@
-# ds-data-registry
-A registry of data sources, categories, and organizations to use with Data Studio Community Connectors.
+# Data Studio Data Registry
+A registry of data sources, categories, and organizations to use with [Data Studio Community Connectors](https://developers.google.com/datastudio/connector/overview).
+
+The registry contains the following :
+- sources/ - Data source definitions and associated metadata.
```
Comment (diminishedprime) [inline README.md]: could you make the path's `sources/`, etc. instead of sources/

File: README.md [hunk 3]
```diff
@@ -1,2 +1,99 @@
...
+| iconUrl           | A link to an icon image representing this source. Recommended dimensions are 40px by 40px. If you need to include the image as part of the pull request; use the sources/icons directory.
+| sourceUrl         | A link users can follow to learn more about the source.
+| dataVisibility    | Valid values: `PUBLIC`, `PRIVATE`. Describes whether the source returns data that is generally available to all users or private. This is not related to authentication or access to data but only about the availability of the data. For example, the data visibility for a Twitter source that returns the number of [googleanalytics](https://twitter.com/googleanalytics) followers is PUBLIC since the source returns the same data for any user. However, for a source such as Google Analytics, the data returned is unique and private to each user so data visibility is PRIVATE. A simple test is if two unique users connected to the source would they receive the same data (PUBLIC) or unique data specific to their account (PRIVATE).
```
Comment (diminishedprime) [inline README.md]: All instances of PUBLIC and PRIVATE in this should probably we wrapped with backticks. `PUBLIC` and `PRIVATE`

File: README.md [hunk 4]
```diff
@@ -1,2 +1,99 @@
...
+
+| Property          | Description |
+| --------          | ------------ |
+| id                | A unique Id. This is used in the [`sources`](https://developers.google.com/datastudio/connector/manifest#sources_ref) property of Community Connector manifests. |
+| name              | The user-friendly name of the source. |
+| categories        | A list of Category Ids. This is used to associate the source with general categories such as Marketing, Advertising, etc. Category Ids must be defined in the [***categories.json***](https://github.com/googledatastudio/ds-data-registry/blob/master/categories.json) registry file.
+| organization      | An organization Id that represents the entity responsible for this source. This is used to associate a source with a top-level organization (E.g. a company or developer). Organization Ids must be defined in the [***organizations.json***](https://github.com/googledatastudio/ds-data-registry/blob/master/organizations.json) registry file.
+| iconUrl           | A link to an icon image representing this source. Recommended dimensions are 40px by 40px. If you need to include the image as part of the pull request; use the sources/icons directory.
```
Comment (diminishedprime) [inline README.md]: Suggestion - don't allow images to be included in this repo.

File: README.md (hunk 3 above)
Comment (diminishedprime) [inline README.md]: change [googleanalytics](https://twitter.com/googleanalytics) to [@googleanalytics](https://twitter.com/googleanalytics)

**Comment by pfrisella** [comment]
Changes made. PTAL.
I'll separate organizations in a different pull request.

**Comment by diminishedprime** [comment]
LGTM. Feel free to go ahead and merge.

**Pull request merged**